rskj-core/src/main/java/co/rsk/peg/union/constants/UnionBridgeConstants.java

```diff
@@ -10,7 +10,7 @@ public class UnionBridgeConstants {
     protected RskAddress unionBridgeAddress;
     protected Coin initialLockingCap;
     protected int lockingCapIncrementsMultiplier;
-    protected AddressBasedAuthorizer changeAuthorizer;
+    protected AddressBasedAuthorizer changeUnionBridgeContractAddressAuthorizer;
```

Comment:
```suggestion
    protected AddressBasedAuthorizer changeAddressAuthorizer;
```
no need for the UnionBridgeContract i think

Reply:
Might be a good idea to review some naming once we have a more complete version

Reply:
Ok. Task created